storage/pruning/pruningStorer.go

```diff
@@ -201,8 +201,12 @@ func initPersistersInEpoch(
 	if oldestEpochActive < 0 {
 		oldestEpochActive = 0
 	}
+	if !args.CleanOldEpochsData {
+		oldestEpochKeep = 0
+	}
 
 	for epoch := int64(args.StartingEpoch); epoch >= oldestEpochKeep; epoch-- {
+		log.Debug("initPersistersInEpoch(): createPersisterDataForEpoch", "identifier", args.Identifier, "epoch", epoch, "shardID", shardIDStr)
 		p, err := createPersisterDataForEpoch(args, uint32(epoch), shardIDStr)
```

Comment:
talking with Iulian, seems that opening and closing 1000 DBs can take up to 20 minutes.
Maybe we can change this function createPersisterDataForEpoch, to only create the reference but not open/close. 
We should only open the active persisters at the end.

Reply:
Sketched a solution.